playersScore.js

```diff
@@ -0,0 +1,131 @@
+const player1Name = document.getElementById("p1-input");
+const player2Name = document.getElementById("p2-input");
+const startBtn = document.getElementById("start-btn");
+const playersSection = document.getElementById("players-container");
+const scoreContainer = document.getElementById("score");
+const restartBtn = document.getElementById("restart-btn");
+
+startBtn.addEventListener("click", startQuiz);
+// Enabling of Start button
+playersSection.addEventListener("change", () => {
+	if (player1Name.value && player2Name.value) {
+		startBtn.disabled = false;
+	}
+});
+
+function startQuiz() {
+	let p1Score = 0;
+	let p2Score = 0;
+	restartBtn.style.display = "block";
+
+	scoreContainer.innerHTML = `
+    <div>
+		<h3 class="pName">${player1Name.value}</h3>
+		<div class="score-buttons">
+        	<button class="incr-btn p1-btn">Correct</button>
+        	<span class="p1Score">${p1Score}</span>
+        	<button class="decr-btn p1-btn">Wrong</button>
+		</div>
+	</div>
```

Comment:
I like the styling of the player scores, and the way that you add them here in JavaScript. 🤩